### PR TITLE
Remove deprecated copy target (on hold)

### PR DESCRIPTION
**Pull request by chfw**
```
copy:
 . ...
```

wont work with 0.6.0 onwards

### CODE REVIEW

File: moban/mobanfile/__init__.py [hunk 1]
```diff
@@ -34,10 +34,9 @@ def handle_moban_file_v1(moban_file_configurations, command_line_options):
 
     targets = moban_file_configurations.get(constants.LABEL_TARGETS, [])
     if constants.LABEL_COPY in moban_file_configurations:
-        legacy_copy_targets = handle_copy(
-            merged_options, moban_file_configurations[constants.LABEL_COPY]
+        raise NotImplementedError(
+            "copy targets has been removed. please refer to level 15 in docs"
```
Comment (ayan-b) [inline moban/mobanfile/__init__.py]: Maybe add the version here? For example `Removed since version ...` and also maybe we can give the alternative.

**Comment by jayvdb** [comment]
Please put this on hold for a while.

coala uses the copy target extensively, and we havent had the chance to upgrade because of the incompatibility of 0.4.x

I believe we can use 0.5.x, and thus can remove the old 'copy:' , but that transition needs to be done yet, and may expose other problems.

**Comment by chfw** [comment]
I can defer this PR later. I am happy that copy has its audience.

My intention was, when people use latest syntax, they may tidy up copy target as well.

File: moban/mobanfile/__init__.py (hunk 1 above)
Comment (chfw) [reply moban/mobanfile/__init__.py]: Without this PR, it does give depreciation warning.

**Comment by chfw** [comment]
This is not intended to get people to worry what will happen next with existing assets. We can always defer the removal. User is the king rather than the developers.

**Pull request closed without merging**